apps/address-service/domains/address/schema/AddressSource.js

```diff
@@ -67,6 +66,16 @@ const AddressSource = new GQLListSchema('AddressSource', {
         },
 
     },
+    kmigratorOptions: {
+        constraints: [
+            {
+                type: 'models.UniqueConstraint',
+                fields: ['source'],
+                condition: 'Q(deletedAt__isnull=True)',
+                name: 'addresssource_source_unique_key',
+            },
+        ],
+    },
```

Comment:
it's the same here

Reply:
The same. I want to restrict only not deleted models.